include/CL/sycl/detail/small_array.hpp

```diff
@@ -95,8 +95,20 @@ struct small_array : std::array<BasicType, Dims>,
     // (*this)[0] is the first element of the underlying array
     std::copy_n(src, Dims, &(*this)[0]);
   }
-
-
+      
+  BasicType& x(){
```

Comment:
Please add 2 lines between each function declaration to have a unified coding style and a Doxygen-style description (starting with either /** or /// ) so that the Doxygen documentation can be built ( see https://xilinx.github.io/triSYCL/Doxygen/triSYCL/html/ for the motivation )